run-tests-editmode-osx-editor.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -eo pipefail
+
+EDITOR_VERSION="2017.4.33f1"
```

Comment:
Can we combine this with the `UNITY_PATH` in the yaml? Or at least define it there so that we only have to change in one file?

Reply:
Going to remove it from the yml for now.